zookeeper-docs/src/main/resources/markdown/zookeeperAdmin.md

```diff
@@ -1797,67 +1796,6 @@ taken to prevent unauthorized access. Such measures are external to
 ZooKeeper (e.g., control access to the files) and depend on the
 individual settings in which it is being deployed.
 
-<a name="Recovery+-+TxnLogToolkit"></a>
-
-#### Recovery - TxnLogToolkit
```

Comment:
Nit: Perhaps we could keep a link to zookeeperTools/txnLogToolkit here, so if anyone is used to having it here can find it easily.

Reply:
That's OK